src/backend/accounts.nim

```diff
@@ -130,6 +173,10 @@ proc isAlias*(value: string): bool =
   let r = Json.decode(response, JsonNode)
   return r["result"].getBool()
 
+proc getRandomMnemonic*(): RpcResponse[JsonNode] {.raises: [Exception].} =
+  let payload = %* []
+  return core.callPrivateRPC("accounts_getRandomMnemonic", payload)
```

Comment:
you could use the template rpcCall to save some code, it basically generate this code